package.json

```diff
@@ -22,6 +22,6 @@
     "sha3": "^1.1.0"
   },
   "browser": {
-    "sha3": "browserify-sha3"
+    "index.js": "browser.js"
```

Comment:
Ok, changed it accordingly. I think it would make sense to support the more advanced constructs of the `browser` field in `package.json`.  Not sure about others, but plenty of blockchain related modules use it that way.